### PR TITLE
 - added DMARC (opendmarc) support with basic setup

### LINKED ISSUE #66: Add DMARC support
As DKIM and SPF should work now in this image, we can look at [DMARC](https://dmarc.org/) support.
I have working setup in my local image, just need time to send a pull request.

### PR DESCRIPTION
**Pull request by bilak**
resolve #66

### CODE REVIEW

File: postfix/default-opendmarc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# /etc/opendmarc.conf. See opendmarc(8) for a complete list of options.
+#DAEMON_OPTS=""
+#
+# Uncomment to specify an alternate socket
+# Note that setting this will override any Socket value in opendkim.conf
+#SOCKET="local:/var/run/opendmarc/opendmarc.sock" # default
+#SOCKET="inet:54321" # listen on all interfaces on port 54321
+#SOCKET="inet:12345@localhost" # listen on loopback on port 12345
+#SOCKET="inet:12345@192.0.2.1" # listen on 192.0.2.1 on port 12345
+SOCKET="inet:54321@localhost"
```
Comment (tomav) [inline postfix/default-opendmarc]: Missing new line.

File: postfix/opendmarc.conf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+PidFile /var/run/opendmarc.pid
+RejectFailures false
+Syslog true
+UMask 0002
+UserID opendmarc:opendmarc
+IgnoreHosts /etc/opendmarc/ignore.hosts
+HistoryFile /var/run/opendmarc/opendmarc.dat
```
Comment (tomav) [inline postfix/opendmarc.conf]: Missing new line.

**Comment by tomav** [comment]
@bilak, could you also add integration tests?

**Comment by tomav** [comment]
And documentation! :-)

**Comment by bilak** [comment]
@tomav I don't know how to add integration tests. In this case I don't even know what to check in logs or elsewhere. 
What do you want to have in documentation? I followed [this](https://www.skelleton.net/2015/03/21/how-to-eliminate-spam-and-protect-your-name-with-dmarc) guide to setup dmarc.

**Comment by tomav** [comment]
If there's nothing to configure / adapt, there's probably nothing to document except that DMARC is available in this image. I'll add it to `README.md` later.

**Comment by tomav** [comment]
Thank you!

**Pull request merged**